cpp/ps0008/problem. 8.cpp

```diff
@@ -23,12 +27,14 @@ std::string decrypt(const std::string& text, int key) {
             result[i] = text[i];
         }
     }
-
     return result;
 }
```

Comment:
- [x] defined function takes two parameters (ciphertext and key)for decryption process
- [x] considered the case of the letter in the message while decryption
- [x] skip numbers and other special characters while decryption
- [x] considered that the key may be positive or negative value
- [x] returned/printed the actual message (message before encryption)